Workouts/HomeView.swift

```diff
@@ -25,15 +25,19 @@ struct HomeView: View {
         return NavigationView {
             List {
                 if featuredWorkout != nil {
-                    NavigationLink(destination: WorkoutDetail(workout: featuredWorkout!)) {
-                        VStack(alignment: .leading) {
-                            Text("Your latest workout 🏅")
-                                .padding(.vertical)
-                                .font(.system(size: 21, weight: .medium))
+                    Section(header: VStack {
+                        Text("Your latest workout 🏅")
+                            .padding(.all)
+                            .font(.system(size: 21, weight: .medium))
+                            .frame(width: UIScreen.main.bounds.width, alignment: .leading)
+                    }) {
+                        ZStack {
                             FeaturedWorkout(workout: featuredWorkout!)
-                                .padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+                            NavigationLink(destination: WorkoutDetail(workout: featuredWorkout!)) {
+                                EmptyView()
+                            }
                         }
-                    }.padding(.trailing, -22.0) // <-- hide that dang little arrow icon
+                    }
```

Comment:
@emilyCheroske Found a hack online for hiding the disclosure indicator that seems a little better: https://nukedbit.dev/hide-disclosure-arrow-indicator-on-swiftui-list/.

And now it doesn't interfere with the map view.